iceoryx2/tests/service_static_config_tests.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#[cfg(test)]
+mod service_static_config_message_type_details {
+    use iceoryx2::service::static_config::message_type_details::{TypeDetail, TypeVariant};
+    use iceoryx2_bb_testing::assert_that;
+
+    #[test]
+    fn test_internal_new() {
+        struct Tmp;
+        let got = TypeDetail::__internal_new::<Tmp>(TypeVariant::FixedSize);
```

Comment:
Please rename `got` and `want` here also to `sut` and `expected`

Reply:
get it, but could i know what's the meaning of `sub`, i haven't thought a useful explanation.